pkg/util/winutil/users.go

```diff
@@ -75,3 +76,50 @@ func GetLocalSystemSID() (*windows.SID, error) {
 
 	return localSystem, err
 }
+
+// GetServiceUserSID returns the SID of the specified service account
+func GetServiceUserSID(service string) (*windows.SID, error) {
+	// get config for datadogagent service
+	user, err := GetServiceUser(service)
+	if err != nil {
+		return nil, fmt.Errorf("could not get datadogagent service user: %s", err)
+	}
+
+	username, err := getUserFromServiceUser(user)
+	if err != nil {
+		return nil, err
+	}
+
+	if username == "LocalSystem" {
+		return windows.StringToSid("S-1-5-18")
+	}
```

Comment:
say why this is needed

```suggestion
        // Manually map some aliases that SCM uses and are not recognized by the
        // security subsystem (`LookupAccountName()` will fail)
        // https://learn.microsoft.com/en-us/windows/win32/services/service-user-accounts
	if username == "LocalSystem" {
		return windows.StringToSid("S-1-5-18")
	}
```